pghrep/templates/H001.tpl

```diff
@@ -4,34 +4,56 @@
 
 ### Master DB server is `{{.hosts.master}}` ###
 
-{{ range $index_name, $index_data := (index (index (index .results .hosts.master) "data") "indexes") }}
-**Index name: `{{ $index_name }}`**
-
-    Reason: {{ $index_data.reason }}
-    Scheme name: {{ $index_data.schemaname }}
-    Table name: {{ $index_data.tablename }}
-    Index size: {{ $index_data.index_size }}
-    Table size: {{ $index_data.table_size }}
-    {{ if $index_data.drop_code }}Drop code: {{ $index_data.drop_code }} {{ end }}
-    {{ if $index_data.revert_code }}Revert code: {{ $index_data.revert_code }} {{ end }}
+#### Indexes ####
+
+Index name | Reason | Scheme name | Table name | Index size | Table size
```

Comment:
it will be shown always. You can just put at the top of the query and get a result "as is":

```
\pset tuples_only
\pset format aligned
```

Reply:
An example:
https://github.com/postgres-ai/postgres-health-check/blob/master/resources/checks/G002_current_connections.sh

Reply:
Oh... it is not psql column names. Looks good.